.gitignore

```diff
@@ -3,3 +3,6 @@ go-mkopensource
 
 # `go mod vendor`
 /vendor/
+
+# Jetbrains IDE
+.idea/
```

Comment:
You really should be sticking that type of thing in your `~/.config/git/ignore`.